terraform/example/main.tf

```diff
@@ -1,38 +1,235 @@
+variable "cert_path" {}
+variable "key_path" {}
+variable "root_ca_path" {}
+
 terraform {
   required_providers {
     teleport = {
       version = "0.0.1"
-      source = "gravitational.com/teleport/teleport"
+      source  = "gravitational.com/teleport/teleport"
     }
   }
 }
 
 provider "teleport" {
-    addr = "localhost:3025"
-    cert_path = "/Users/xnutsive/go/src/github.com/gravitational/teleport-plugins/docker/tmp/auth.crt"
-    key_path = "/Users/xnutsive/go/src/github.com/gravitational/teleport-plugins/docker/tmp/auth.key"
-    root_ca_path = "/Users/xnutsive/go/src/github.com/gravitational/teleport-plugins/docker/tmp/auth.cas"
+  addr         = "localhost:3025"
+  cert_path    = var.cert_path
+  key_path     = var.key_path
+  root_ca_path = var.root_ca_path
+
+  # addr = "evilmartians.teleport.sh:443"
+  # identity_file_path = "id"
```

Comment:
Remove commented code?

Reply:
I've added additional comments to this commented lines.